changelog.txt

```diff
@@ -1,3 +1,12 @@
+2.7.0
+=============
+- **Debugging into MSAL was not possible**. MSAL now supports source link. This means developers can debug into MSAL, using VS 15.7+, which will allow them to generate a snukpg package that contains the pdb files (one per platform) and they can add a manifest containing urls to source files in the nuget package. Use the NuGet package explorer to see this manifest. [MSAL PR](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/750)
```

Comment:
MSAL integrates SourceLink https://github.com/dotnet/sourcelink. This allows MSAL to embed pdb files and source code in the NuGet package, allowing users to debug into MSAL without replacing their package reference with a project reference.

Reply:
thanks, @bgavrilMS